script/build.sh

```diff
@@ -106,9 +106,13 @@ function docker_cmd()
         echo "building ${DOCKERHUB_REGISTRY}:${HASH_TAG}..."
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} .
 
+        echo "scanning docker image..."
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --severity CRITICAL "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```

Comment:
Recommendation:
```suggestion
        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/trivy-cache:/root/.cache/ aquasec/trivy --severity CRITICAL --no-progress "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```

This way, we have a consistent cache location that's OS agnostic (and we don't try to show an animated progress bar that doesn't print nicely)

Reply:
I also added `--exit-code 1` to the suggestion, as we're going with that for our other images.